apps/tenant-management-webapp/src/app/pages/admin/services/comment/topics/deleteConfirmationsView.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FunctionComponent, useEffect, useState } from 'react';
+
+import { useDispatch } from 'react-redux';
+
+import { TableDiv } from '../styled-components';
+import { DeleteModal } from '@components/DeleteModal';
+import { deleteTopicRequest } from '@store/comment/action';
+
+import { TopicItem } from '@store/comment/model';
+
+interface calendarTableProps {
+  topic: TopicItem;
+  selectedType: string;
+  onCancel?: () => void;
+  onDelete?: (topic) => void;
+}
+
+export const DeleteConfirmationsView: FunctionComponent<calendarTableProps> = ({
```

Comment:
Hi @RamaChandiniDevarapalli , I think we can use the better way to pass the properties here. Otherwise, I feel we will do another around code refactoring in the near future. And, the FunctionComponent is something we need to avoid.